packages/components/src/components/appheader/appheader.stories.ts

```diff
@@ -0,0 +1,46 @@
+import type { Meta, StoryObj, Args } from '@storybook/web-components';
+import '.';
+import { html } from 'lit';
+import { classArgType, styleArgType } from '../../../config/storybook/commonArgTypes';
+import '../button';
+import '../brandvisual';
+import '../chip';
+import '../avatar';
+
+const render = (args: Args) => html`
+  <mdc-appheader args.property="${args.property}"></mdc-appheader>`;
+
+const meta: Meta = {
+  title: 'Components/appheader',
+  tags: ['autodocs'],
+  component: 'mdc-appheader',
+  render,
+  parameters: {
+    badges: ['stable'],
+  },
+  argTypes: {
+    ...classArgType,
+    ...styleArgType,
+  },
+};
+
+export default meta;
+
+export const Example: StoryObj = {
+  render: () => html`
+    <mdc-appheader>
+      <div slot="leading" style="display: flex; align-items: center; gap: 8px">
+        <mdc-button variant="tertiary" size="32" color="default" prefix-icon="application-panel-bold"
```

Comment:
```suggestion
        <mdc-button variant="tertiary" size="32" prefix-icon="application-panel-bold"
```
tertiary buttons do not have color